src/main/java/com/salesforce/mirus/config/SourceConfigDefinition.java

```diff
@@ -33,6 +33,12 @@
       "",
       ConfigDef.Importance.HIGH,
       "Java regex whitelist.  Matching topics names will be mirrored"),
+  TOPICS_REGEX_LIST(
+      "topics.regex.list",
+      ConfigDef.Type.LIST,
+      Collections.EMPTY_LIST,
+      ConfigDef.Importance.HIGH,
+      "Java regex list.  Matching topics names will be mirrored"),
```

Comment:
I think you should day "Comma separated Java regex list. Matching topics names will be mirrored. Regex strings must not contain commas."

Reply:
While I think of it, we probably should deprecate "topics.whitelist" and change to  "topics.allowlist". Doesn't have to be in this PR though.

Reply:
Thanks, updated the description.